src/js/classes/Graph.js

```diff
@@ -232,47 +242,38 @@ export class ScatterPlot {
           r: this.getNumber(r[key])
         };
       });
-      points.push(new CountyPoint(data, ...countyInfo));
-    }
+      return new CountyPoint(data, ...countyInfo);
+    });
 
-    const sortedPoints = points.sort((a, b) => b.data[0].r - a.data[0].r);
-    return sortedPoints;
+    return points.sort((a, b) => b.data[0].r - a.data[0].r);
   }
 
   updateViewBox() {
     const prevSizing = this.sizing;
     this.sizing = getSizing(window.innerWidth);
+    const isRegularWidth = this.sizing === REGULAR_WIDTH;
 
     // set viewbox based on window size (customized for specific phones)
     const width =
       this.sizing === SMALL_PHONE
-        ? 180
+        ? 250
         : this.sizing === LARGE_PHONE
         ? 280
         : this.sizing === SMALL_BROWSER
         ? 300
         : 600;
-    const height = this.sizing === REGULAR_WIDTH ? 500 : 400;
+    const height = isRegularWidth ? 500 : 400;
     this.plot.setAttributeNS(null, "viewBox", `0 0 ${width} ${height}`);
     if (prevSizing !== this.sizing) {
-      // rerender axes with mobile sizing value
-      this.renderAxis(this.xAxis, false);
-      this.renderAxis(this.yAxis, true);
-      //set radii with mobile sizing value
-      this.points.forEach((point) => {
-        const circles = point.elements.filter(
-          (el) => el instanceof SVGCircleElement
-        );
-        circles.forEach((_, i) => {
-          circles[i].setAttributeNS(
-            null,
-            "r",
-            this.sizing === REGULAR_WIDTH
-              ? point.rsDesktop[i]
-              : point.rsMobile[i]
-          );
-        });
-      });
+      // adjust axis ticks and labels for mobile/desktop
+      const dxy = isRegularWidth ? 26 : 18;
```

Comment:
I see 26 & 18 in a couple places so that might make sense to convert to constants. Doesn’t block this PR, likely just a follow up.

Reply:
Yeah good point. Honestly I don't know where those numbers originated from; I guess it's just the spacing that looked good when we were initially coding up the graphs. Either way constants make sense!